### ISSUE TITLE
"Overrides to check" - for non Developers

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-cms#42749]
### Is your feature request related to a problem? Please describe.
There is a feature that tracks changes for Overrides. However if you use a 3rd Party template without adjustments I think the template provider should have the possibilty to add a flag or something to the override that its checked by them?! It just does not make any sense that I check the templates from someone else?
### Describe the solution you'd like
Information in the override for which version it was checked, maybe with a code comment or so 
Joomla reads this and does not alarm

### Additional context
Think of end users that think something is wrong, when the template provider could actually flag themselves that its checked

**Comment by brianteeman** [comment]
As we already see many template providers offering single downloads with no/limited updates then this will be a problem.

**Example.** 
I buy a template from provider x who marks all of their overrides as checked (why wouldnt they) 
I no longer have support from the template provider so I dont get any more updates
Joomla makes updates and you as the site owner will never know about it because your template providert marked it all as ok

How do we resolve that?

**Comment by coolcat-creations** [comment]
You mark them checked for Version 5.0.2 for example. Its up to the template provider to offer the service or not. Also Template Providers have the bonus that they can say, that the template NEEDs to be updated.

**Comment by brianteeman** [comment]
_I am just thinking out aloud_

I buy a template from provider x who marks all of their overrides as checked for 5.0.2
I make some changes to those overrides
Joomla releases 5.0.3
Joomla tells me I have template overrides to check
Template developer issues update marked as ok for 5.03
I install that update and lose all my changes

**Comment by coolcat-creations** [comment]
If you update your template  yourself, you always lose your changes. That has nothing to to with that issue.
Actually my problem here is that as a customer from a provider I have not clue what to adjust in their code.
Maybe the flag should be lost when a user edits the override. Thats true. Because then the guarantee by the template provider can't be held anymore.

**Comment by brianteeman** [comment]
_real world example_

Every template created for Joomla 4 that has overrides for com_content or com_contact needs to be updated for Joomla 5. So for a template provider to mark their template overrides as checked they will need to maintain at least two versions of every template (while joomla 4 is still supported)

**Comment by coolcat-creations** [comment]
Ok so the overrides are not correct for Joomla 5 anyway - then this needs to be adressed anyway or not ?

**Comment by brianteeman** [comment]
_real world example_

I just checked a leading template provider and they only offer one download for their templates and it is only fully compatible with joomla 5 but there is no mention of it not supporting joomla 4

I checked another leading template provider and they only offer one download for their templates and it is only fully compatible with joomla 4 etc

**Comment by coolcat-creations** [comment]
Yes I understand you, but then there is a general issue or not?
If you get the mention to check 44 Overrides from a template provider, do you contact them, or what do you do in that case?

**Comment by brianteeman** [comment]
> Yes I understand you, but then there is a general issue or not?

yes there is a general issue but I dont see it as resolved by this proposal.

The first part of the issue is that Joomla did not communicate the major changes in template layouts between j4 and j5. If it had done that then at least part of your issue is easily resolved. _hint_ search for all instances of itemprop and remove them

The second part is that I don't trust template providers to  be 100% truthful about their templates compatibility. Based on a 2 second check of two leading providers they aren't now so why should they be in the future.

> If you get the mention to check 44 Overrides from a template provider, do you contact them, or what do you do in that case?

Personaly I would check them 

Finaly one reason that there are often such a large number to check is that when creating an override using joomla it will create an override for every file even if you dont change it. Personaly I think it is good practice after creating overrides to use the difference function and delete all the unchanged files. That massively reduces the number os files to check.

For example on one of my sites I have an override to change blog links to a ul from an ol BUT by default that will produce 8 files to check in the future.

**Comment by Quy** [comment]
Related/duplicate  #38653?
